site/content/blog/2025-02-17-portswigger-lab-walkthrough-broken-brute-force-protection-ip-block/index.md

```diff
@@ -0,0 +1,755 @@
+---
+title: "PortSwigger Lab Walkthrough: Broken Brute-Force Protection, IP Block"
+summary: >
+  Walkthrough for the PortSwigger lab, "Broken brute-force protection, IP block".
+images:
+- https://www.zaproxy.org/blog/2025-02-17-portswigger-lab-walkthrough-broken-brute-force-protection-ip-block/images/image header-key and house models.jpg
```

Comment:
@kingthorin I followed the example of the other articles here. Is this correct?

Reply:
Yeah, I'd suggest using underscores or hyphens in the filename, and reducing the length (it seems overly verbose).
(Here and the file itself obviously :wink:)

Reply:
> reducing the length (it seems overly verbose)

Just to be clear, you mean just the filename of the image, yes? Not the entire article (that one's long, too)?

Reply:
Ya just the image